scipy/signal/_fir_filter_design.py

```diff
@@ -895,7 +895,11 @@ def firls(numtaps, bands, desired, weight=None, nyq=None, fs=None):
     bands : array_like
         A monotonic nondecreasing sequence containing the band edges in
         Hz. All elements must be non-negative and less than or equal to
-        the Nyquist frequency given by `nyq`.
+        the Nyquist frequency given by `nyq`. The bands are specified as
+        frequency pairs, thus, if using a 1D array, its length must be
+        even, e.g., `np.array([0, 1, 2, 3, 4, 5])`. Alternatively, the
+        bands can be specifiedas an nx2 sized 2D array, where n is the
```

Comment:
`specifiedas` -> two words I think?

Also, I think I don't see the doc build running in CI, perhaps CircleCI doesn't repeat the reporting if it runs the test on your `main` branch or something. Unfortunately, I can't push that change in to your `main`.

I think we can get this merged really soon, you may see me remove the milestone to keep the upcoming release checklist focused on the most critical things, but just ping me when that is fixed and we can merge/restore the milestone.

Reply:
Oops, nice catch @tylerjereddy!

Seems like force pushing didn’t trigger the CI last time. I think CI have now run. 

One test failed, but I am not sure if it is due to my code changes.